dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -539,6 +539,25 @@ html, body {
   }
 }
 
+.headerlink:link, .headerlink:visited {
+  color: white;
+  text-decoration: none;
+}
+
+.headerlink:hover, .headerlink:active {
+  color: white;
+  border-bottom: solid 4px rgb(255, 138, 0);
+  background-color: initial;
+}
+
+.headerlink {
```

Comment:
should we use text-transform: uppercase instead of making the actual text uppercase?